lib/batch/fix_series_iv_827.rb

```diff
@@ -0,0 +1,17 @@
+# 827 Series IV Photographs Collection Changes
+# Move all records in each Series IV subcollection to Series IV collection,
+# and delete each subocllection.
+
+series_iv_collection = Collection.find("x346d4254")
+
+series_iv_collection.members.each do |subcollection|
+  if subcollection.class == Collection
+    subcollection.members.each do |member|
+      if member.class == GenericFile
```

Comment:
What happens if there are other sub-collections within the sub-collection? E.g Collection -> Sub-collection1 -> [GenericFile1, GenericFile2, Sub-collection2 -> [GenericFile3, GenericFile4]]

Reply:
That's a good question. Normally I would use recursion here in case the collections do continue to have other collections. For this particular one though, Series IV Photographs, there are no additional collection beyond this level so I didn't worry about adding recursion.

Reply:
I figured that might be the case. Thanks for following up on this and glad you had already considered it.